src/containers/Application.js

```diff
@@ -445,14 +445,14 @@ class Application extends Component {
 
 	addHelpButton(path) {
 		useEffect(() => {
-			this.HelpService.setData(path);
+			this.HelpService.setPath(path);
```

Comment:
@Aringoaway Do we actually need the HelpService then? we can store the path directly to the store I assume

```
		this.Store.dispatch({
			type: HELP_CONTENT,
			path: path
		});
```

Reply:
I agree. That's a great idea.